tests/test_kubo_cas.py

```
@@ -0,0 +1,146 @@
+    Requires a local IPFS daemon on default ports.
+    """
+    # Check if local IPFS daemon is available on default ports
+    import http.client
+
+    try:
+        conn = http.client.HTTPConnection("127.0.0.1", 5001, timeout=1)
+        conn.request("POST", "/api/v0/version")
+        response = conn.getresponse()
+        if response.status != 200:
+            pytest.skip("No IPFS daemon running on default ports (127.0.0.1:5001)")
+    except Exception:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)